### PR TITLE
Update importer contract to make it compatible with exporter

### PR DESCRIPTION
**Pull request by eaypek-tfh**
**Change**
- This PR makes sure the contract between importer and exporter are the same
- It sets the necessary configs for buckets
- It adds the logic to avoid overriding already loaded records from DB with S3 records

### CODE REVIEW

File: iris-mpc-store/src/s3_importer.rs [hunk 1]
```diff
@@ -95,12 +95,17 @@ fn hex_to_bytes(hex: &str, byte_len: usize) -> eyre::Result<Vec<u8>> {
 }
 
 pub async fn last_snapshot_timestamp(store: &impl ObjectStore) -> eyre::Result<i64> {
-    store
-        .list_objects()
-        .await?
+    let objects = store.list_objects().await?;
+    tracing::info!("All objects in db chunks s3: {:?}", objects);
```
Comment (philsippl) [inline iris-mpc-store/src/s3_importer.rs:99]: do we really want to log them all?

File: iris-mpc-store/src/s3_importer.rs [hunk 2]
```diff
@@ -112,7 +117,8 @@ pub async fn fetch_and_parse_chunks(
     let chunks = store.list_objects().await.unwrap();
     stream::iter(chunks)
         .filter_map(|chunk| async move {
-            if chunk.ends_with(".bin") {
+            if chunk.ends_with(".csv") {
+                tracing::info!("Processing chunk: {}", chunk);
```
Comment (philsippl) [inline iris-mpc-store/src/s3_importer.rs:121]: same here

**Review by philsippl — APPROVED** [review]

File: iris-mpc-store/src/s3_importer.rs (hunk 1 above)
Comment (eaypek-tfh) [reply iris-mpc-store/src/s3_importer.rs:99]: let me remove that one 👍

File: iris-mpc-store/src/s3_importer.rs (hunk 2 above)
Comment (eaypek-tfh) [reply iris-mpc-store/src/s3_importer.rs:121]: I'd like to keep this one as it can help with future debugging. it's printing one by one

**Pull request merged**